src/model/data.py

```diff
@@ -117,7 +121,42 @@ def create_loader(IMAGES_PATH, LABEL_PATH, INDEX_SUBSET=None, CLASS_SUBSET=None,
         batch_size=BATCH_SIZE,
         num_workers=num_workers,
         pin_memory=pin_memory,
+        shuffle=False,
     )
     
     return org_loader
 
+
+# A Python generator for adversarial samples.
```

Comment:
update description of function, yields org and adv sample that are classified correctly for the org but incorrectly for the adv sample